app/templates/components/info-form.hbs

```diff
@@ -1,4 +1,4 @@
-<form {{action "save" on="submit"}} class="test-addon-info-form">
+<form {{action save on="submit"}} class="test-addon-info-form">
```

Comment:
I think the `isSaving` property in this file may be missing after the move to components.